settings/AdvancedSettings.cpp

```diff
@@ -31,9 +31,16 @@ void AdvancedSettings::reset()
     m_writeThumbUrlsToNfo = true;
     m_useFirstStudioOnly = false;
 
+    m_videoCodecMappings.insert("v_mpeg4/iso/avc", "h264");
+    m_videoCodecMappings.insert("v_mpegh/iso/hevc", "HEVC");
+    m_videoCodecMappings.insert("27", "h264");
```

Comment:
This doesn't seem correct... Is there any source on mappings like this other than trial-and-error? :)

Reply:
This is what I got from ML:
![1](https://user-images.githubusercontent.com/12618414/42409324-14915d4c-821c-11e8-960a-5071d39f8573.jpg)
![2](https://user-images.githubusercontent.com/12618414/42409327-182ed3c6-821c-11e8-8a41-c939a86814a4.jpg)
Etc.

Reply:
Thanks. But I think mappings like "27" to "H264" are something the user should add to `advancedsettings.xml` on his own. I just can't find any resource that says that H264 may be encoded as "27" (neither official nor inofficial). I'm also unsure whether this might be a bug by the program used to encode your video files. 
This includes `("2", "MPEG2")`, `("234", "VC-1")` and `("27", "h264")`. 

I'm fine with all other codecs (even though I I couldn't find and source on `microsoft` as well). :smiley: